tree/ntuple/v7/test/ntuple.cxx

```diff
@@ -663,6 +663,7 @@ TEST(RNTuple, RDF)
    EXPECT_EQ(42.0, *rdf.Min("pt"));
 }
 
+constexpr unsigned int RNTupleDescriptor::kNBytesPostscript;
```

Comment:
I got a linker error here, so I included a declaration as in https://stackoverflow.com/questions/8452952/c-linker-error-with-class-static-constexpr . Maybe it has to do with the fact that I use a C++14 compiler (Clang) and not C++17...

Reply:
Thanks, good catch!  It should be taken care of by #4291

Reply:
Should this line then be removed?